lib/rubocop/cop/variable_force/branch.rb

```diff
@@ -226,6 +226,21 @@ def always_run?
           end
         end
 
+        # case target
+        # in clause # in_clause
```

Comment:
How about the name `in_pattern`?

```suggestion
        # in patter # in_pattern
```

c.f.:

```ruby
% ruby-parse -e 'case foo; in bar; end'
(case-match
  (send nil :foo)
  (in-pattern
    (match-var :bar) nil nil) nil)
```

Reply:
Sounds good!